.circleci/config.yml

```diff
@@ -207,6 +207,7 @@ jobs:
             pip install --user --progress-bar off mypy
             pip install --user --progress-bar off --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
             pip install --user --progress-bar off --editable .
+            mypy --install-types
```

Comment:
you'll need to edit config.yml.in instead and call `python .circleci/regenerate.py` (it's a weird system, yup ;)   )

Reply:
Should we have a workflow that checks that both are in sync?

Reply:
I think that's the role of `circleci_consistency`

Reply:
Currently we have a job called `circleci_consistency` which checks for that.